### PR TITLE
Use assertNull

### PR DESCRIPTION
**Pull request by carusogabriel**
I've refactored some tests using the [`assertNull`](https://phpunit.de/manual/current/pt_br/appendixes.assertions.html#appendixes.assertions.assertNull) method.

### CODE REVIEW

**Comment by SvenRtbg** [comment]
Good thing!

How's anyones thought about `assertEquals` (compares with `==`) vs. `assertSame` (compares with `===`). I do prefer the latter and try to avoid the former if at all possible because the same rule also applies to the production code.

File: tests/UnitTest/FunctionsTest.php [hunk 1]
```diff
@@ -152,7 +152,7 @@ public function test_env_default_value_null()
         $this->assertTrue($definition instanceof EnvironmentVariableDefinition);
         $this->assertEquals('foo', $definition->getVariableName());
         $this->assertTrue($definition->isOptional());
-        $this->assertSame(null, $definition->getDefaultValue());
```
Comment (mnapoli) [inline tests/UnitTest/FunctionsTest.php]: I'm not sure `assertSame()` should be replaced here, it's not exactly the same thing. IIRC `assertNull` does a loose comparison (`== null`) which isn't the same as a strict one (`assertSame`).

Comment (carusogabriel) [reply tests/UnitTest/FunctionsTest.php]: @mnapoli Actually no, is a strict comparison. See [this PHPUnit method](https://github.com/sebastianbergmann/phpunit/blob/master/src/Framework/Constraint/IsNull.php#L27) that checks this conditional.

Comment (mnapoli) [reply tests/UnitTest/FunctionsTest.php]: Awesome TIL

Thanks!

**Pull request merged**